build/ci/Dockerfile

```diff
@@ -7,19 +7,38 @@ ENV KUBECTL_VERSION=1.13.2
 ENV DOCKER_VERSION=18.03.1-ce
 ENV GOLANGCILINT_VERSION=1.17.1
 
-# Download kubebuilder release to get required tools (etcd, apiserver, etc.)
-ENV PATH=${PATH}:/usr/local/kubebuilder/bin
-RUN curl -fsSLO https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${KUBEBUILDER_VERSION}/kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz && \
-    tar -zxf kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz && \
-    mv kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64 /usr/local/kubebuilder
-
 # Download required golang tools
 RUN go get github.com/golang/dep/cmd/dep golang.org/x/tools/cmd/goimports
 
 # Install golangci-lint
 RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh \
     | sh -s -- -b $(go env GOPATH)/bin v${GOLANGCILINT_VERSION}
 
+# Add Go dependencies to Docker image
+WORKDIR /go/src/github.com/elastic/cloud-on-k8s/operators
+COPY operators/Gopkg.lock .
+COPY operators/Gopkg.toml .
+RUN dep ensure --vendor-only -v
+
+WORKDIR /go/src/github.com/elastic/cloud-on-k8s/local-volume
+COPY local-volume/Gopkg.lock .
+COPY local-volume/Gopkg.toml .
+RUN dep ensure --vendor-only -v
+
+# Cleanup
+RUN rm /go/src/github.com/elastic/cloud-on-k8s/local-volume/Gopkg.lock && \
+    rm /go/src/github.com/elastic/cloud-on-k8s/local-volume/Gopkg.toml && \
+    rm /go/src/github.com/elastic/cloud-on-k8s/operators/Gopkg.lock && \
+    rm /go/src/github.com/elastic/cloud-on-k8s/operators/Gopkg.toml
+WORKDIR /
```

Comment:
It is a good habit to put at the end of a Dockerfile the part that evolves most often in order to benefit from the Docker build cache. That's not very relevant here in the CI context but it is if we build the image locally.

Reply:
Fixed